local-volume/provisioner/pkg/deleter/deleter_test.go

```diff
@@ -171,8 +172,9 @@ func testSetup(t *testing.T, config *testConfig) *Deleter {
 
 	config.apiUtil = util.NewFakeAPIUtil(config.apiShouldFail, config.cache)
 	userConfig := &common.UserConfig{
-		MountDir: testMountDir,
-		HostDir:  testHostDir,
+		DiscoveryMap: map[string]common.MountConfig{
+			"sc1": common.MountConfig{HostDir: testHostDir + "/test-dir", MountDir: testMountDir + "/test-dir"},
```

Comment:
nit: can you put HostDir and MountDir on separate lines to make it more readable?